server/src/utils/Logger.ts

```diff
@@ -93,7 +93,7 @@ export class ConnectionLogger implements Logger {
     description: string,
     callback: () => Promise<unknown>
   ) {
-    this.info(`${description}: Start`);
```

Comment:
Still seeing log output for compilation:

![image](https://user-images.githubusercontent.com/24030/201331170-18be530c-bfb8-4ad3-a043-a27a6a01492c.png)

I think these are coming from `logger.trackTime`

Reply:
:man_facepalming: changed the "start" line to trace and missed the "end" one. Sorry!